src/controllers/post.js

```diff
@@ -20,6 +21,30 @@ export const create = async (req, res) => {
   return sendDataResponse(res, 201, { post: createdPost })
 }
 
+export const createComment = async (req, res) => {
+  const { content, postId } = req.body
+
+  if (!content) {
+    return sendDataResponse(res, 400, { content: 'Must provide content' })
+  }
+  const createdComment = await dbClient.postComment.create({
```

Comment:
You might want to add a try/catch block here? See the user controller for examples - you want to catch any errors from the database call and return a 500 response.